data/strings/types_strings.txt

```diff
@@ -7298,7 +7142,8 @@
     zh-Hant = 美式餐廳
 
   [type.cuisine.donut]
-    en = Donut
+    en = Doughnut
+    en-US = Donut
```

Comment:
I don't think that en-US is supported at the moment. `en` is US by default, everything else should be added as an exception. There was an issue about that, so we should refactor our code.

Reply:
So en = Donut, en_GB = Doughnut?